.github/workflows/test-code.yml

```diff
@@ -71,11 +71,16 @@ jobs:
         if: ${{ startsWith(matrix.os, 'windows') }}
         working-directory: src
         run: dotnet test --no-restore --no-build
-      - name: Test AOT compilation
-        working-directory: src/GraphQL.AotCompilationSample
+      - name: Test code-first AOT compilation
+        working-directory: src/GraphQL.AotCompilationSample.CodeFirst
         run: |
           dotnet publish -c Release -o publish
```

Comment:
```suggestion
          dotnet publish -c Release -o publish &&
```

Reply:
It's already been tested and works correctly the way I wrote it.

Reply:
Because without `&&` exit error code would be the code of the last command run. I understand that the second command is unlikely to be completed (even started properly) if the first ended with an error, because we have fresh builds every time, but who knows (caches, bugs, unforseen conditions)? I already had to deal with such errors earlier because of the forgotten `&&` operator in scripts performing several commands sequentially.

Reply:
While this may be true for script files, it does not apply in this case.  GitHub Actions processes each command independently and will stop if any one fails.

See sample run:
- https://github.com/graphql-dotnet/graphql-dotnet/actions/runs/3953265287/jobs/6769348325

![image](https://user-images.githubusercontent.com/6377684/213305668-8acb7d6b-967a-46ad-bff3-ab409a421482.png)

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun

Reply:
Strange. Fine. it just works, OK.